src/ResourceManager/Network/Commands.Network/NetworkWatcher/SetAzureNetworkWatcherConfigFlowLogCommand.cs

```diff
@@ -91,13 +132,93 @@ public class SetAzureNetworkWatcherConfigFlowLogCommand : NetworkWatcherBaseCmdl
         [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
         public SwitchParameter AsJob { get; set; }
 
+        [Alias("EnableTA")]
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
```

Comment:
Remove ValueFromPipelineByPropertyName for this parameter (doesn't make sense for SwitchParameters).

Reply:
Done.

Reply:
Still can this be given a second thought? Since form the name of the parameter sometimes people assume it to be a boolean and try to specify $true/$false values. ValueFromPipelineByPropertyName=true will take them in account as I understand. And the existing network cmdlets are also using ValueFromPipelineByPropertyName=true in switch parameters. Please advise here.

Reply:
Can we give it a second thought? Sometime people assume it to be a boolean field and try to use as " -EnableTrafficAnalytics $true"...actually I see the "ValueFromPipelineByPropertyName = true" is being used in the other network watcher cmdlets for switch parameters. Please advise here.